solution/ui/regulations/css/scss/_application_settings.scss

```diff
@@ -189,6 +189,22 @@ $policy_subject_close_background_hover: $mid_gray_3;
         format("truetype");
 }
 
+@font-face {
+    font-family: "Merriweather";
+    font-style: normal;
+    font-weight: 400;
+    src: url("#{$additional-font-path}/Merriweather-Regular.ttf")
+        format("truetype");
+}
+
```

Comment:
We had apparently not been rendering Merriweather fonts on the site?  So this fixed that.

Reply:
nice catch